cli/src/stake.rs

```diff
@@ -1881,6 +1887,15 @@ pub fn process_delegate_stake(
     }
 }
 
+pub fn is_stake_program_v2_enabled(rpc_client: &RpcClient) -> bool {
+    rpc_client
+        .get_account(&feature_set::stake_program_v2::id())
+        .ok()
+        .and_then(|account| feature::from_account(&account))
+        .and_then(|feature| feature.activated_at)
+        .is_some()
```

Comment:
`RpcClient::is_feature_enabled(feature_id)` might be a useful method someday, but just have this specific use for now.

Reply:
@CriesofCarrots Oh, sorry for post merge comment, but, I think this might be better off checking the current slot of cluster. Otherwise, there is small time window where this returns too newer stake calculation behavior when the feature is pending to be activated for the next epoch.

Reply:
I believe this is correct as-is. The activation_at field in the account is changed from `None` to `Some(slot)` on the epoch boundary when the feature activates: https://github.com/solana-labs/solana/blob/master/runtime/src/bank.rs#L4119-L4122
When the feature activation has been requested, but is still pending, L1895 returns `None`, so the old stake calculation is used.

This is the same logic used here: https://github.com/solana-labs/solana/blob/master/cli/src/feature.rs#L298

Reply:
oh yeah. you're right! Sorry for misguiding you... I misunderstood the code. It seems that my in-brain runtime feature activation code got corrupted somehow... ;)